README.md

```diff
@@ -54,50 +54,47 @@ Project is designed to utilize django's powerful features.
 
 ### Getting Your Environment Set Up:
 
-**This project was initialized with python 3.11.3.**
-Please be sure when building the front end repository that you have the correct versions of npm and node.
-The correct version can be found and installed [here](https://nodejs.org/en/download/current).
+**This project was initialized with python 3.11.3.** The download for which can be found [here](https://www.python.org/downloads/release/python-3113/).
+Please be sure when building the front end repository that you have the correct versions of node and npm. The correct version can be found and installed [here](https://nodejs.org/en/blog/release/v20.2.0).
 
 To check if your versions are correct you can use the following commands:
-Check node version with `node -v` -> This should show v20.2.0 in the console
-Check npm version with `npm -v` -> This should show 9.6.7 in the console
-
-1.  Clone and change directories to the repository in the directory (ex. Desktop) you would like to work in. `git clone [repo link]`.
-    Development is done on the `dev` branch. Ensure you are working either on `dev` or a branch that has branched off of `dev`.
-
-- To Switch to `dev` or another branch, `git checkout branch_name` ex: `git checkout dev` will switch to branch.
-
-2. Install the correct version of python & pip.
-3. Install pipenv with pip -> `pip install pipenv`
-4. Initialize the virtual environment for yourself and installs virtual environment dependencies -> `pipenv install`
-**IMPORTANT** Get the .env file from someone and store it on your root directory '/SITE' if you haven't already before running this command.
-5. Activate the virtual environment -> `pipenv shell`
-6. Go to `Site/meshapp/` and run an `npm install`
-7. Build the frontend -> in the same directory as step 7 run `npm run build`
-8. Return to the root directory and start the backend server / run the application -> `python manage.py runserver`
-MYSQL SETUP (windows, if you have a mac, consider switching to a windows)
-1. Download Mysql over here https://dev.mysql.com/downloads/installer/ ( i downloaded the file that was 331.3 M)
-2. Do the setup and do the full download (spam next until you see the password portion) 
-3. Make sure you are not using the legacy version, select the 8.x version
```

Comment:
why was this removed?

Reply:
This wasn't removed. I reformatted it.

Reply:
Maybe im blind where is the "make sure you are not using legacy version"

Reply:
You're right I did remove it, I'll add it back in